### PR TITLE
Add a new korath Hand to Hand weapon

### PR DESCRIPTION
**Pull request by Brick63**
adds a rifle to replace the intrusion countermeasures

### CODE REVIEW

**Comment by ghoulavenger** [comment]
I object to the illegality of the rifle. Illegal terms only apply to humans who make laws and scan ships as part of enforcement. Unless the Quarg are enforcing the illegality of these outfits it doesn't make sense to make them illegal.

As was part of the Deep Archaeology questline suggested, "There was no need to make it illegal, because nobody could do it." Although this would be more like "There was no need to make it illegal, because we didn't know about it." I have no comment on the actual stats though, looks slightly better than nerve gas, granted a lot more annoying to acquire.

**Comment by themightygrunt** [comment]
Personal Opinion on it? It's alright but only necessary if you want to remove the Korath having Quarg gear otherwise meh, just introduces issues.

**Comment by Pointedstick** [comment]
Making it more effective than nerve gas is going a bit overboard. Making it illegal may also be unnecessary: nerve gas an atrocity because it's a chemical weapon; throughout most of the history of warfare, even the most horrific conventional weapons have been considered fair game. I mean heck, frag grenades are legal...

I love the idea of alien hand-to-hand weapons though and support this weapon's inclusion, even if it requires some tweaks.

**Comment by Brick63** [comment]
there is historical distaste for munitions that are designed to cause more harm to people so that's one reason for the illegality

**Comment by Brick63** [comment]
another is to balance the strength versus the other options as if it were legal then players could get these with no repercussions

**Comment by ghoulavenger** [comment]
>there is historical distaste for munitions that are designed to cause more harm to people so that's one reason for the illegality

Are you suggesting that these guns use torture rounds or something? Honestly it's a weapon designed to kill somebody, that makes it no worse than a regular rifle. My point still stands though, illegal requires an authority that can make such a judgment. It won't be the humans (because of lack of knowledge), and probably not the pug (because of their distaste of the Quarg/Drak). Which leaves the Kor Efret (which don't have the manpower to do the enforcement, not to mention they're a pretty peaceful lot under the protection of the Quarg), and the Quarg/Drak (which might oppose them on principle, but aren't likely to enforce laws upon them, only intervene when they are destroying themselves or others). I suppose the Wanderers might object? But I don't think enough to interfere with the sovereignty of the Korath Outcasts (since they're a peaceful lot by nature). By your words, they're perfectly legal among the Korath Outcasts, so they wouldn't do the enforcement either. So where does the illegality come from?

> another is to balance the strength versus the other options as if it were legal then players could get these with no repercussions

You're using a mechanic to justify it being OP in your book? I think you don't understand the point of the mechanic -- look at the quarg ships for example, Quarg Skylances are ridiculously gamebreakingly OP. They aren't illegal. But if you really do think theres a balance issue, then you should lower their effectiveness instead imho.

**Comment by Brick63** [comment]
im trying to make it balanced and as for the skylances those are T3 this isnt anywhere near that strong also yes there are munitions that are banned because they do more damage incendiary is the closest one to point at
also as to whos enforcing the illegality NG works the same if the planet sees you have it you get a fine all races do it

**Comment by Pointedstick** [comment]
Grammar please.

And don't forget to submit HiDPI images at https://github.com/endless-sky/endless-sky-high-dpi.

**Comment by ghoulavenger** [comment]
> yes there are munitions that are banned because they do more damage incendiary is the closest one to point at

Huh? The only weapons in game currently that are currently illegal are nerve gas and I think nukes (could bee wrong on that one). Everything else that is illegal are contraband from pirate systems and the like.

**Comment by Brick63** [comment]
I'm speaking about real life laws as there is nothing comparable to this in the game which states that any munitions that are designed for the express purpose of doing more damage are Banned from use. surely those laws would still be in place as Chemical weapons still are. also the Syndicate clearly know that the Korath exist and have enough sway in parliament that they could easily make these illegal to use.

**Comment by ghoulavenger** [comment]
> I'm speaking about real life laws as there is nothing comparable to this in the game which states that any munitions that are designed for the express purpose of doing more damage are Banned from use. surely those laws would still be in place as Chemical weapons still are. 

I'm not sure what laws you're referencing, but granted the existence of illegal arms, the argument may be valid, assuming the lore supports it (there are currently no cases in game that do, but that could change). However it still fails to address my argument that humans (the ones that currently enforce all laws) couldn't possibly create laws for them because they don't know about them. Nor would any of the alien species actively enforce laws on them if they had that capacity.

> the Syndicate clearly know that the Korath exist and have enough sway in parliament that they could easily make these illegal to use.

Referencing the FW campaign, the existence of the Korath is known by the Syndicate and top officials in the parliament. But the general populace did not -- the general populace might know about the Quarg, but the other alien species are far more elusive. Making these weapons illegal for the general populace would need to explain the source of these weapons and possibly require efforts on the part of parliament to combat the Korath menace. No, it makes far more sense to me that parliament would ignore them and let the Syndicate handle it on their own. 

And thats without even considering the limited access to the tech itself. I mean the Parliament didn't even know the Syndiate had jump drives, so why would the Syndicate detail out weapons for them? In fact, I'd wager the Syndicate might try to reproduce such weapons instead if they had them rather than try and get their potential products made illegal -- theres just no profit in that.

And if ES was anything like reality, it's far more likely that people would be trying to steal the tech from you to study it, than fine you for having it -- which would definitely be open season on pretty much all alien outfits, with the potential exception being the Hai (since humans and Hai do co-exist peacefully and offer their technology to humans).

**Comment by tehhowch** [comment]
My thoughts:
1) Drop illegal, until we are able to make the illegality of outfits depend on where you are. When we can do that, we can make this outfit illegal to the Quarg and Kor Efret (and Archons too, perhaps). As mentioned above, humanity has pretty tolerant standards for weaponry.
2) make this outfit very hard to get unless you already have it
   - Keeping the 1 on the Chaser provides the only "easy" way to get it
   - Korath Raiders should be fully equipped with these weapons. They're raiding parties after all, armed to the teeth. This will make Raiders much harder to capture, until the player has farmed up a number of rifles from capturing Chasers and then has crew number advantage with tech level parity.
   - Korath World-Ships should have at least 200 (lored as not every crew member on a world ship is going to be trained military who should have these)
3) Adjust the actual number on World-Ships such that the capture odds for a fully-crewed Bactrian (465 bunks) fully kitted KRR's is the same as it is now using Nerve Gas vs the Intrusion Countermeasures.

An alternate thought is to impart some sort of utility cost to these weapons, such as active energy or heat production, though this would then require compensating the AI ships which have these outfits.

**Comment by Pointedstick** [comment]
Excellent ideas, @tehhowch. I fully agree.

**Comment by Brick63** [comment]
now how many would that be for the Worldship then as i dont know the numbers and would if left alone give the WS 500 and call it a day

also for the raider do you mean full total crew or full required crew

**Comment by Brick63** [comment]
ill just put it at 466 for now to mess with people until i get a good number

**Comment by tehhowch** [comment]
for the Raider, give it `Required + 1/2 (Total - Required)`
for the Worldship just do some trial and error battle comparisons unless you want to do some math. Set up the 465 nerve gas baccy vs a derelict worldship and use the combat strategy from **[here](http://steamcommunity.com/app/404410/discussions/0/2381701715711029856/#c2381701715710373997)**
Note the capture chances at 50% crew and 25% crew, then adjust the number of KRR until those same targets are hit with the KRR loadout.

File: data/korath outfits.txt [hunk 1]
```diff
@@ -310,3 +310,12 @@ outfit "Steering (Stellar Class)"
 	"turning energy" 8.2
 	"turning heat" 26.7
 	description "After millennia of war, the Korath have developed some truly enormous warships. These steering systems give even their largest warships a reasonable amount of agility in combat."
+	
+outfit "Korath Repeater Rifle"
+	category "Hand to Hand"
+	cost 220000
```
Comment (Pointedstick) [inline data/korath outfits.txt]: 220,000 credits for a single rifle seems a little ridiculous--especially considering that you can't buy these, so players will be able to sell them for quite a pretty penny.

Comment (Pointedstick) [inline data/korath outfits.txt]: 220,000 credits for a single rifle seems a little ridiculous--especially considering that you can't buy these, so players will be able to sell them for quite a pretty penny, considering the numbers with which they appear.

File: data/korath outfits.txt [hunk 2]
```diff
@@ -310,3 +310,12 @@ outfit "Steering (Stellar Class)"
...
+	thumbnail "outfit/korath rifle"
+	"capture attack" 3.1
+	"capture defense" 5.4
+	"unplunderable" 1
+	description "The Korath Repeater Rifle, while commonplace among the Korath, is considered highly illegal by everyone but the lawless who see them as trophies of sorts and will pay nicely for it."
```
Comment (Pointedstick) [inline data/korath outfits.txt]: Since it's no longer illegal, this is no longer accurate.

File: data/korath outfits.txt (hunk 1 above)
Comment (Brick63) [reply data/korath outfits.txt]: gives the players something to do with the extras

File: data/korath outfits.txt (hunk 2 above)
Comment (Brick63) [reply data/korath outfits.txt]: look i will make it illegal once i can choose who its illegal for but till then everyone loves it

File: data/korath outfits.txt (hunk 1 above)
Comment (Amazinite) [reply data/korath outfits.txt]: I'd lower this to at most 160,000 credits per rifle, but closer to 125,000 per rifle seems like a better number.

**Comment by Amazinite** [comment]
>Drop illegal, until we are able to make the illegality of outfits depend on where you are. [Tehhowch]

I don't see why illegal should be dropped until then. This weapon is more powerful than nerve gas, therefore making nerve gas completely obsolete if this weapon isn't illegal. A substantially higher illegal value makes an interesting balance point. "Do I take the nerve gas or do I take the weapon that is only VERY slightly more powerful but WAY more expensive in terms of the fines I'll get."

>Korath Raiders should be fully equipped with these weapons. They're raiding parties after all, armed to the teeth. This will make Raiders much harder to capture, until the player has farmed up a number of rifles from capturing Chasers and then has crew number advantage with tech level parity [Tehhowch]

A full equipped Raider would gain (5.4 * 250 =) 1350 defense from this. They currently have only 240 defense from their 4 intrusion countermeasures. As long as a Raider can still be captured, I'm sure it's fine, but just looking at it right now that seems pretty extreme.

>I object to the illegality of the rifle. Illegal terms only apply to humans who make laws and scan ships as part of enforcement. Unless the Quarg are enforcing the illegality of these outfits it doesn't make sense to make them illegal. [ghoulavenger]

Then on what terms is the illegality of nerve gas justified, as all factions will fine you for nerve gas just as all factions would fine you for this weapon.

>Are you suggesting that these guns use torture rounds or something? [ghoulavenger]

Discussion during development came to the conclusion that since this weapon is based off of the Repeater, which does heat damage, the rounds of the gun would be incendiary. Part of the modern laws of war involve protecting combatants from unnecessary suffering. Incendiary ammunition can be argued to deal unnecessary harm through the burning that is left behind by the bullet. Therefore, this weapon would be illegal. It would just appear that mention of incendiary rounds didn't make it into the description.

In terms of balance though, I think it's necessary for a weapon stronger than nerve gas to be illegal so that nerve gas doesn't become obsolete, at least in this stage in the game's development. MZ may think differently though.

File: data/korath outfits.txt (hunk 2 above)
Comment (Amazinite) [inline data/korath outfits.txt]: `The Korath Repeater Rifle, while commonplace among the Korath, is considered highly illegal by more civilized societies due to the unnecessary suffering its incendiary ammunition may cause to its target. `

Comment (Brick63) [reply data/korath outfits.txt]: me likey

**Comment by ghoulavenger** [comment]
> Then on what terms is the illegality of nerve gas justified, as all factions will fine you for nerve gas just as all factions would fine you for this weapon.

Humans and other societies can make any weapons they want illegal. My argument was entirely from a lore perspective with the assumption that only humans penalize you for illegal items -- as of right now, they are the only ones that scan and fine you to my knowledge. If they aren't, then I would argue that *nerve gas* should be judged differently on their illegality scale as well *unless* there is a substantiation for the illegality -- like the Coalition might ban *all weapons* until you have a license for them (since they don't sell weapons to civilians, and there is no threat within their borders anyway, you coming in armed as a macho man might be perceived as a threat that needs to be addressed, especially if you start arming their ship frames or are hauling loose weapons in cargo with intent to distribute -- which I see as a potential mission).

> Discussion during development came to the conclusion that since this weapon is based off of the Repeater, which does heat damage, the rounds of the gun would be incendiary. Part of the modern laws of war involve protecting combatants from unnecessary suffering. Incendiary ammunition can be argued to deal unnecessary harm through the burning that is left behind by the bullet. Therefore, this weapon would be illegal. It would just appear that mention of incendiary rounds didn't make it into the description.

I ceded that this argument may be valid as long as it held true within the lore of the game, for which there is no current *precedent*. But it still does *not* justify it being illegal in human space, since this is *not* something you would find in human space, nor do most people within human space even know about the Korath, so there is still no *lore* to support this type of ban. Which was my point. Illegal is used as a *lore* theme not a *balance* theme. If it was based on *balance*, then a lot of outfits would be very much so illegal. I'm perfectly fine with you objecting to a stronger hand to hand weapon than nerve gas though. Because that actually is *balance*, and I'm perfectly fine with nerve gas being the strongest available hand to hand weapon, there isn't really a need for more diverse weapons because it's very unlikely to change the nature of the capturing game more than simply having more personnel -- nerve gas is 2.8, this was proposed 3.1, which is roughly 10% stronger, the equivalent of going from 245 crew to 270 crew or so. Which won't make a big difference if you have to drop 70 or so crew on the first ship you capture (which is approximately a 30% strength drop immediately assuming 245 crew, regardless of what weapon is being used), as long as you aren't opening capturing of higher options up -- which is mostly Korath and Quarg ships.

In any case I don't believe that anything in game has or will ever be balanced on its illegality. Nerve gas certainly isn't. If you don't want to get in trouble with nerve gas, you just stay in the southern rim, pirate worlds, and most Syndicate worlds. Considering the easiest way to take advantage of nerve gas is in the southern worlds anyway where pirates are at their strongest concentrations and enforcement at their lowest the illegality of nerve gas is an annoyance at best, this would not change if you were to make this new hand to hand weapon illegal.

**Comment by Amazinite** [comment]
>as of right now, they are the only ones that scan and fine you to my knowledge.

Humans and Captain Pester scan. All races will fine upon landing with illegal outfits.

>I ceded that this argument may be valid as long as it held true within the lore of the game, for which there is no current precedent.

Then let's make the precedent now: incendiary hand to hand weapons are now illegal.

>But it still does not justify it being illegal in human space... so there is still no lore to support this type of ban

In the same vein, there is no lore reason that all factions should fine you for having nerve gas, but that's a matter of #1429. Given that such a mechanic does not exist, we shouldn't be forced to limit ourselves in terms of what we can create and how it can be created. And I would argue that whether humanity knows about the Korath is irrelevant. If your ship is being searched on landing then the people doing the searching don't need to know about the Korath to know that this is a rifle, and hey look, here's the incendiary ammunition for it. Have a fine. In terms of scans, the Navy might not know of the Korath, but enough merchants hanging around the core do, and pirates do as well. It's reasonable to assume that in the 100+ years that the Korath have been raiding the Core, some Repeater rifles found their way into Navy hands, and they know search for them in their scans and hand out fines.

**Comment by ghoulavenger** [comment]
> In the same vein, there is no lore reason that all factions should fine you for having nerve gas, but that's a matter of #1429. Given that such a mechanic does not exist, we shouldn't be forced to limit ourselves in terms of what we can create and how it can be created. 

We're going to have to agree to disagree then. I think you're misusing the mechanic, you think you're using it as intended.

> In terms of scans, the Navy might not know of the Korath, but enough merchants hanging around the core do, and pirates do as well. It's reasonable to assume that in the 100+ years that the Korath have been raiding the Core, some Repeater rifles found their way into Navy hands, and they now search for them in their scans and hand out fines.

One does not hand out fines without telling you what they were for, and such laws are posted within a community. Such a thing would be advertising the existence of the Korath to the general populace. Right now the only race that humanity sponsors contact with is the Quarg, after the FW campaign the pug (which conveniently leaves), and pretty much shuts their mouth up about the Hai (which are just a hop skip and a jump through some uninhabited systems.) There might be some information about the Korath during the Wanderer campaign depending on which options you choose, but most of that tech is unrecoverable anyway since they're drone ships (which means there wouldn't be the rifles anyway either). What we can assume is that information is being controlled, very certainly.

Merchants and pirates are a potential problem, but most merchants aren't warriors, especially ones capable of boarding Korath vessels and pirates are rooted out by the Syndicate in that region of space. The Syndicate doesn't stand to make a profit by handing over anything to the Republic.

**Comment by Amazinite** [comment]
>One does not hand out fines without telling you what they were for, and such laws are posted within a community. 

It would be known that incendiary weapons are illegal.
The weapon itself shows the illegal attribute.

>Such a thing would be advertising the existence of the Korath to the general populace. 

Again, one does not need to know of the Korath to know what a rifle is and to figure out what this specific rifle does (i.e. fire incendiary ammunition). A simple ban on incendiary hand to hand weaponry would suffice in order to ban this weapon. The law isn't going to say "The use of Korath Repeater Rifles is prohibited," it'll say "The use of incendiary weaponry in hand to hand combat is prohibited."

>Merchants and pirates are a potential problem, but most merchants aren't warriors, especially ones capable of boarding Korath vessels and pirates are rooted out by the Syndicate in that region of space. The Syndicate doesn't stand to make a profit by handing over anything to the Republic.

Again, 100+ years of Korath raids. That's plenty of time for just a single rifle to make its way to the Republic/Navy. They don't even need to know what it is, they can just think it's some strange experimental pirate weapon. And the uninhabited systems in the Far North give any pirate attacking Korath ships in Durax the ability to leave the Core and travel into Navy patrolled space.

**Comment by solardawning** [comment]
Bear in mind, adding a stronger offensive hand to hand weapon means that Quarg Skylarks and Wardragons become easily capturable. (Though this feels like less of an issue now that Quarg anger is permanent, and presumably will prevent the player from doing future storylines)

**Comment by Brick63** [comment]
skylances are take-able as you can plunder them and a bacy with NG can cap a wardragon

**Comment by Amazinite** [comment]
The Skylark can already be capped fairly easily, and the Wardragon isn't so much better than the Skylark that it's worth capturing, let alone a worry in terms of making slightly stronger H2H weapons..

**Comment by Brick63** [comment]
oh skylark not lance oops

**Comment by KiLEdEnNis** [comment]
my only concern (and I'm sure others' too) is that boarding is gonna become extremely easy, making capping high-end ships a big quick money grind, although unless ya like to piss off the Coalition or something ya have to stick to pirates & Unfettered Hai
and by the time ya even get enough rifles (and getting them at the start is very hard) ya should already be quite end-game anyway, so it shouldn't be that bad

**Comment by Brick63** [comment]
My main goal with this is to replace the quarg outfits on korath ships past that I'll take suggestions for changes

**Comment by KiLEdEnNis** [comment]
I don't really have good suggestions, and higher tier ships are bound to have better H2H outfits
just like how say T2 tech beats human tech, T2 H2H should beat human ones, plus ya achieved the goal of replacing the intrusion countermeasures
the ONLY problem this has is the fact boarding is an OP money-making strategy, but that's not really a valid reason to not make any H2H at all
we just have to find a way to make boarding balanced, ya did a good job w/ this PR

**Comment by Nescio0** [comment]
> adds a rifle

No objections, new items are welcome.

> to replace the intrusion countermeasures

Why is that necessary? There is nothing wrong with those, they work and are interesting.

**Comment by tehhowch** [comment]
Quarg tech.

Adds a bit of lore, too - for a group of exiled aliens whose only joy in life is to crush their enemies, see them driven before them, and to hear the lamentations of their women, they certainly are lacking in the offensive raiding weaponry department.

**Comment by endless-sky** [comment]
It doesn't make sense to me for this weapon to be so much more powerful than human hand-to-hand weapons. Something more along the lines of 1.5 - 1.8 attack and 2 - 3 defense would be more reasonable.

**Comment by Brick63** [comment]
okay changes were made

**Comment by Amazinite** [comment]
>It doesn't make sense to me for this weapon to be so much more powerful than human hand-to-hand weapons.

Maybe if looking at the Repeater Rifle vs the human Laser Rifle (as the Repeater Rifle would have about 5 times the attack), but in terms of the Repeater Rifle vs Nerve Gas it isn't "so much more powerful" (only about 10.7% more attack)

**Comment by Kryes-Omega** [comment]
I think about it this way, Nerve gas is a gas grenade, so the (very deadly) gas is going to spread through rooms, hallways, etc, but the Repeater rifle is a rifle, which isn't going to have the crowd-clearing effect that the gas would, as well as being higher risk to the user because you have to poke your head out to shoot them rather than tossing it in and waiting. So no matter how powerful it is compared to the laser rifle, I wouldn't say that it should be better than the Nerve gas.
_Unless,_ it takes up some outfit space to become more of a heavier weapon with more effect, sort of like the attack version of the security station. Then it would become an interesting alternative to the standard Nerve gas, trading more attack power for outfit space.

**Comment by KiLEdEnNis** [comment]
should it really be an illegal weapon though? it's basically a T2 rifle
if the Korath got some T2 equivalent to Nerve Gas, that item should be illegal, but not the rifle in my eyes
adding that it's hard to even acquire these rifles since ya have to cap a raider (although once ya do ya get a big boost), it should pay off a bit

**Comment by Brick63** [comment]
Well it's illegal in the sense that it uses plasma and that is considered to cause undue suffering.

**Comment by KiLEdEnNis** [comment]
a bit realistic yes, but unnecessary balance-wise to me, as it's just defensive Nerve Gas but 1 tier higher

**Comment by jafdy** [comment]
As it stands these are worse for the player as a H2H weapon then nerve gas, and harder to obtain. Either the values need to be put back up or the illegal needs to be dropped.

File: data/korath outfits.txt [hunk 3]
```diff
@@ -313,10 +313,9 @@ outfit "Steering (Stellar Class)"
...
 	category "Hand to Hand"
-	cost 69000
+	cost 73000
 	thumbnail "outfit/korath rifle"
 	"capture attack" 1.6
 	"capture defense" 2.4
-	"illegal" 620000
 	"unplunderable" 1
-	description "The Korath Repeater Rifle, while commonplace among the Korath, is considered highly illegal by more civilized societies due to the unnecessary suffering its incendiary ammunition may cause to its target."
+	description "This rifle is commonplace among Korath ship crews and predates the creation of the Repeater used on Korath ships by at least two centuries. Since it uses superheated plasma instead of a laser it is slow to fire and will sometimes burn the users hands if not careful"
```
Comment (KiLEdEnNis) [inline data/korath outfits.txt]: the end feels a bit clunky (opinion), and ya forgot the period after the sentence

Comment (Brick63) [reply data/korath outfits.txt]: thanks for noticing that and yeah i agree on the clunk of the end

File: data/korath outfits.txt [hunk 4]
```diff
@@ -318,4 +318,4 @@ outfit "Korath Repeater Rifle"
 	"capture attack" 1.6
 	"capture defense" 2.4
 	"unplunderable" 1
-	description "This rifle is commonplace among Korath ship crews and predates the creation of the Repeater used on Korath ships by at least two centuries. Since it uses superheated plasma instead of a laser it is slow to fire and will sometimes burn the users hands if not careful"
+	description "This rifle is commonplace among the Korath and predates the creation of the Repeater by at least two centuries. By using superheated plasma instead of lasers, The rifle is slow to fire and can sometimes burn the users hands if they aren't careful while making up for it by almost guaranteeing those hit won't live long enough to fire back."
```
Comment (KiLEdEnNis) [inline data/korath outfits.txt]: as said on Discord:
"The rifle" while there was a comma before "The"
and the last sentence is way too long

Comment (Brick63) [reply data/korath outfits.txt]: thanks again

**Comment by Brick63** [comment]
one question i have is if this should be sold by the efret now that it isnt so powerful or if i should leave it as something thats needs to be farmed from the exiles

**Comment by KiLEdEnNis** [comment]
I'd say let the players work for it; don't sell it, since it's already a good buff
PLUS if the Efreti are so peace-loving, why would they sell it

**Comment by Brick63** [comment]
Well i was thinking that as they will use them on their WSs it would only be right to sell them but i see the point

File: copyright [hunk 5]
```diff
@@ -257,6 +257,11 @@ Files: images/outfit/tactical?scanner*
 Copyright: Zachary Siple
 License: CC-BY-SA-4.0
 
+Files: images/outfit/korath?rifle*
+Copyright: Evan Fluharty (Evanfluharty@gmai.com)
+License: CC-BY-SA-4.0
+Comment: Derived from works by Michael Zahniser(From under the same license).
```
Comment (Bladewood) [inline copyright]: `Zahniser(From`

File: copyright [hunk 6]
```diff
@@ -257,6 +257,11 @@ Files: images/outfit/tactical?scanner*
 Copyright: Zachary Siple
 License: CC-BY-SA-4.0
 
+Files: images/outfit/korath?rifle*
+Copyright: Evan Fluharty (Evanfluharty@gmai.com)
+License: CC-BY-SA-4.0
+Comment: Derived from works by Michael Zahniser (From under the same license).
```
Comment (Bladewood) [inline copyright]: Mate, I said to lowercase the F.

Comment (Brick63) [reply copyright]: oops

**Pull request merged**